src/test/java/uk/ac/cam/cl/dtg/isaac/api/UsersFacadeIT.java

```diff
@@ -0,0 +1,86 @@
+package uk.ac.cam.cl.dtg.isaac.api;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpSession;
+import jakarta.ws.rs.core.Response;
+import org.junit.Before;
```

Comment:
We shouldn't be using JUnit 4.

Reply:
Unfortunately, updating the Befores in this test file appears to require updating the base IT test, which leads to updating all the other tests dependant on that. While that's doable, I've already done (most of) that in another ticket in review (#24), so preferably we'd merge that one in and update this one to match rather than doing it twice?